### DISCUSSION TITLE
My github page and localhost server doesn't look the same

### DISCUSSION THREAD

**Discussion by lidormaymon** [community/community#50819] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

So basically in my localhost server my web looks like that as i want it to be
https://gyazo.com/c632f0de72d73584a27217d682a376c9
And on github page it looks like that
https://lidormaymon.github.io/iPhone-Caculator/

Any clue what could be the reason for that and how to fix it?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @lidormaymon ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
